MANIFEST.in

```diff
@@ -2,6 +2,7 @@ include AUTHORS
 include Gruntfile.js
 include INSTALL
 include LICENSE
+include license.python.txt
```

Comment:
If you rebase your branch, you'll notice I moved this file in a separate commit.